docs/guides/akamai/get-started/parent-child-accounts/index.md

```diff
@@ -0,0 +1,154 @@
+---
+slug: parent-child-accounts
+description: 'Learn how parent and child accounts can help Akamai partners manage multiple accounts.'
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+modified: 2024-04-23
+keywords: ["akamai partners", "parent", "child", "parent/child relationship"]
+modified_by:
+  name: Linode
+published: 2022-04-23
+title: Parent and Child Accounts for Akamai Partners
+aliases: ['/guides/akamai/get-started/parent-child-account/']
+authors: ["Linode"]
+promo_default: false
+---
+## Overview
+The parent and child accounts feature was designed with Akamai partners and their clients in mind. It allows partners to manage multiple end customers’ accounts with a single login to their company’s account.
+
+{{< note type="secondary" >}}
+This feature is available only for Akamai partners and their end customers with an Akamai Cloud Computing contract. To learn more about it, contact your Akamai representative.
```

Comment:
Also, who has the Akamai Cloud Computing contract... isn't it the Akamai partner? So, maybe this would read better as:

"This feature is only available to Akamai partners with a Cloud Computing contract and their end customers."

Reply:
no, end customers need to have Akamai Cloud Computing contract